README.md

```diff
@@ -9,52 +7,50 @@
 
 ## Overview
 
-### Introduction
-
-*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrates to users a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
+*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrate a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
 
 Configure the data ingestion from SIEM using connectors such as Elasticsearch or Splunk. The hunting playbooks then trigger the hunt for the specified period mentioned in the hunt record and create an alert for the corresponding MITRE Technique.
 
-Refer to the following Simulation Scenarios' to experience the use case without any SIEM configuration.
+Refer to the following Simulation Scenarios to experience the use case without any SIEM configuration.
 
-### Usage
+## Usage
 
-This Solution Pack ships with the following simulation scenarios. [Refer](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to Simulate Scenario documentation to understand how to Simulate and Reset Scenario.
+This Solution Pack ships with the following simulation scenarios. Refer to [Simulate Scenario documentation](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to understand how to Simulate and Reset scenarios.
 
```

Comment:
We can add "This Solution Pack ships with the following simulation scenarios." here just before the scenrion details starts...